src/commands/deploy/nextjs/deploy.ts

```diff
@@ -409,40 +423,78 @@ function writeNextConfig(cwd: string, region: string) {
     if (!existingConfig) {
         const extension = determineFileExtension(cwd);
         writeConfigFiles(cwd, extension, region);
-        return;
     }
 
-    const configPath = path.join(cwd, `next.config.${existingConfig}`);
-    const handlerPath = `./cache-handler.${existingConfig}`;
+    const currentConfigPath = path.join(cwd, `next.config.${existingConfig}`);
+    const baseNextConfigPath = path.join(cwd, `base-next.${existingConfig}`);
+
+    // Rename next.config.{ext} to base-next.{ext}
+    fs.renameSync(currentConfigPath, baseNextConfigPath);
+
+    const genezioConfigPath = path.join(cwd, `next.config.${existingConfig}`);
```

Comment:
This seems to be the same with `currentConfigPath` - maybe instead of `currentConfigPath` and `baseNextConfigPath` use `genezioConfigPath` and `userConfigPath`.